cardano-node/src/Cardano/Node/Configuration/TopologyP2P.hs

```diff
@@ -288,6 +292,34 @@ readPeerSnapshotFile  (PeerSnapshotFile peerSnapshotFile) =
   where
     handleException = handleAny $ \e -> error $ "Cardano.Node.Configuration.TopologyP2P.readPeerSnapshotFile: " <> displayException e
 
+defaultTopology :: [a] -> NetworkTopology a
+defaultTopology addresses = RealNodeTopology
```

Comment:
Is this applicable to mainnet too? If not, I think we should keep it in `cardano-testnet`.